### DISCUSSION TITLE
Menu component in ultima layout

### DISCUSSION THREAD

**Discussion by luksmk** [primefaces/community#676] [PrimeFaces Templates] [answered]
Hi,
Is there a reason why menu component is no longer packaged in ultima jar file ?
Up until version 5 it used to be bundled in jar with static files. From version 6 only resource files are packaged.
If that was intentional change, what is a proper way to include ultima layout with menu into my project ?

Thanks

**Comment by mertsincan** [comment, marked as answer]
Hi,

Yes, most of the users want to override it, and having a java file in the jar has caused big problems, especially for those who want to use Jakarta or those who have switched to Jakarta later. We received a lot of questions about this issue and we decided to let users add those files manually. You can produce a jar according to your needs with small adjustments in our tag project.

Best Regards,
